modules/ducktests/tests/ignitetest/services/ignite.py

```diff
@@ -38,10 +38,47 @@ def __init__(self, context, config, num_nodes, jvm_opts=None, startup_timeout_se
         super().__init__(context, config, num_nodes, startup_timeout_sec, shutdown_timeout_sec, modules=modules,
                          jvm_opts=jvm_opts)
 
+    @property
+    def database_dir(self):
+        """
+        :return: path to database directory
+        """
+        return os.path.join(self.work_dir, "db")
+
+    @property
+    def snapshots_dir(self):
+        """
+        :return: path to snapshots directory
+        """
+        return os.path.join(self.work_dir, "snapshots")
+
     def clean_node(self, node):
         node.account.kill_java_processes(self.APP_SERVICE_CLASS, clean_shutdown=False, allow_fail=True)
         node.account.ssh("rm -rf -- %s" % self.persistent_root, allow_fail=False)
 
+    def rename_database(self, new_name: str):
```

Comment:
it performs "mv" but has the "rename*" name...

Reply:
"move" implies getting a new path. we rename the base folder

Reply:
Seems you changed `mv` to `cp`, what the reason to perform a so heavy operation?

Reply:
removed this method